build.sbt

```diff
@@ -481,6 +489,106 @@ lazy val unidocSettings =
       (JavaUnidoc / unidoc / unidocAllSources).value.map(apiDocSourceFilter)
   )
 
+lazy val genTunablesDocSettings = Seq(
+  Compile / genTunablesDoc := {
+    val stream = (propgen / streams).value
+    val dafExtFile =
+      (propgen / Compile / resources).value.find(_.getName == "dafext.xsd").get
+    val outputDocFile = (Compile / target).value / "tunables.md"
+    // parse xsd file
+    val dafExtXml = scala.xml.XML.loadFile(dafExtFile)
+    // extract tunables information
+    val tunablesElements =
+      (dafExtXml \ "element").filter(_ \@ "name" == "tunables") \\ "all" \ "element"
+    // build documentation
+    val documentationTuple = tunablesElements.map { ele =>
+      val subtitle = ele \@ "name"
+      val documentation = (ele \ "annotation" \ "documentation").text.trim.replaceAll(" +", " ")
```

Comment:
Suggest doing `(ele \ "annotation" \ "documentation").text.split("\n").map(_.trim).mkString("\n")`. Triming each individual line will avoid a space at the beginning of lines. I *think* the extra space shouldn't affect the markdown since it's just a single space, but for a format that is somewhat space-sensitive it's probably best to ensure there are no leading spaces.